Find a functionally equivalent code:android.widget.CalendarView.getSelectedWeekBackgroundColor:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the background color for the selected week.
 *
 * @return The week background color.
 *
 * @attr ref android.R.styleable#CalendarView_selectedWeekBackgroundColor
 */

@@ -1,3 +1,3 @@
 {
-    return mSelectedWeekBackgroundColor;
+    return mDelegate.getSelectedWeekBackgroundColor();
 }

